Failure to compile using Capstone 3

```
FAILED: librz/analysis/librz_analysis.so.0.5.0.p/arch_x86_x86_il.c.o 
gcc -Ilibrz/analysis/librz_analysis.so.0.5.0.p -I. -I.. -Ilibrz -I../librz -Ilibrz/include -I../librz/include -I../librz/analysis/arch/gb -Ilibrz/util/sdb/src -I../librz/util/sdb/src -Ilibrz/util/sdb -I../librz/util/sdb -I../librz/asm/arch/include -I../librz/asm/arch -I../librz/asm/arch/h8300 -I../librz/asm/arch/hexagon -I../librz/asm/arch/msp430 -I../librz/asm/arch/rsp -I../librz/asm/arch/mcore -I../librz/asm/arch/v850 -I../librz/asm/arch/propeller -I../librz/asm/arch/ebc -I../librz/asm/arch/cr16 -I../librz/asm/arch/8051 -I../librz/asm/arch/v810 -I../librz/asm/arch/or1k -I../librz/bin/format -I../librz/type/parser -I../subprojects/capstone-3.0.5/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Werror -O3 -Wimplicit-fallthrough=3 -DRZ_PLUGIN_INCORE=1 -D_GNU_SOURCE --std=gnu99 -Werror=sizeof-pointer-memaccess -fvisibility=hidden -Wno-cpp -fPIC -MD -MQ librz/analysis/librz_analysis.so.0.5.0.p/arch_x86_x86_il.c.o -MF librz/analysis/librz_analysis.so.0.5.0.p/arch_x86_x86_il.c.o.d -o librz/analysis/librz_analysis.so.0.5.0.p/arch_x86_x86_il.c.o -c ../librz/analysis/arch/x86/x86_il.c
../librz/analysis/arch/x86/x86_il.c:98:3: error: ‘X86_REG_DR8’ undeclared here (not in a function); did you mean ‘X86_REG_R8’?
   98 |  [X86_REG_DR8] = "dr8",
      |   ^~~~~~~~~~~
      |   X86_REG_R8
../librz/analysis/arch/x86/x86_il.c:98:3: error: array index in initializer not of integer type
../librz/analysis/arch/x86/x86_il.c:98:3: note: (near initialization for ‘x86_registers’)
../librz/analysis/arch/x86/x86_il.c:99:3: error: ‘X86_REG_DR9’ undeclared here (not in a function); did you mean ‘X86_REG_R9’?
   99 |  [X86_REG_DR9] = "dr9",
      |   ^~~~~~~~~~~
      |   X86_REG_R9
../librz/analysis/arch/x86/x86_il.c:99:3: error: array index in initializer not of integer type
../librz/analysis/arch/x86/x86_il.c:99:3: note: (near initialization for ‘x86_registers’)
../librz/analysis/arch/x86/x86_il.c:100:3: error: ‘X86_REG_DR10’ undeclared here (not in a function); did you mean ‘X86_REG_R10’?
  100 |  [X86_REG_DR10] = "dr10",
      |   ^~~~~~~~~~~~
      |   X86_REG_R10
../librz/analysis/arch/x86/x86_il.c:100:3: error: array index in initializer not of integer type
../librz/analysis/arch/x86/x86_il.c:100:3: note: (near initialization for ‘x86_registers’)
../librz/analysis/arch/x86/x86_il.c:101:3: error: ‘X86_REG_DR11’ undeclared here (not in a function); did you mean ‘X86_REG_R11’?
  101 |  [X86_REG_DR11] = "dr11",
      |   ^~~~~~~~~~~~
      |   X86_REG_R11
../librz/analysis/arch/x86/x86_il.c:101:3: error: array index in initializer not of integer type
../librz/analysis/arch/x86/x86_il.c:101:3: note: (near initialization for ‘x86_registers’)
../librz/analysis/arch/x86/x86_il.c:102:3: error: ‘X86_REG_DR12’ undeclared here (not in a function); did you mean ‘X86_REG_R12’?
  102 |  [X86_REG_DR12] = "dr12",
      |   ^~~~~~~~~~~~
      |   X86_REG_R12
../librz/analysis/arch/x86/x86_il.c:102:3: error: array index in initializer not of integer type
../librz/analysis/arch/x86/x86_il.c:102:3: note: (near initialization for ‘x86_registers’)
../librz/analysis/arch/x86/x86_il.c:103:3: error: ‘X86_REG_DR13’ undeclared here (not in a function); did you mean ‘X86_REG_R13’?
  103 |  [X86_REG_DR13] = "dr13",
      |   ^~~~~~~~~~~~
      |   X86_REG_R13
../librz/analysis/arch/x86/x86_il.c:103:3: error: array index in initializer not of integer type
../librz/analysis/arch/x86/x86_il.c:103:3: note: (near initialization for ‘x86_registers’)
../librz/analysis/arch/x86/x86_il.c:104:3: error: ‘X86_REG_DR14’ undeclared here (not in a function); did you mean ‘X86_REG_R14’?
  104 |  [X86_REG_DR14] = "dr14",
      |   ^~~~~~~~~~~~
      |   X86_REG_R14
../librz/analysis/arch/x86/x86_il.c:104:3: error: array index in initializer not of integer type
../librz/analysis/arch/x86/x86_il.c:104:3: note: (near initialization for ‘x86_registers’)
../librz/analysis/arch/x86/x86_il.c:105:3: error: ‘X86_REG_DR15’ undeclared here (not in a function); did you mean ‘X86_REG_R15’?
  105 |  [X86_REG_DR15] = "dr15",
      |   ^~~~~~~~~~~~
      |   X86_REG_R15
../librz/analysis/arch/x86/x86_il.c:105:3: error: array index in initializer not of integer type
../librz/analysis/arch/x86/x86_il.c:105:3: note: (near initialization for ‘x86_registers’)
../librz/analysis/arch/x86/x86_il.c: In function ‘x86_il_get_operand_bits’:
../librz/analysis/arch/x86/x86_il.c:876:2: error: enumeration value ‘X86_OP_FP’ not handled in switch [-Werror=switch]
  876 |  switch (op.type) {
      |  ^~~~~~
cc1: all warnings being treated as errors
```

--- a/librz/analysis/arch/x86/x86_il.c
+++ b/librz/analysis/arch/x86/x86_il.c
@@ -95,6 +95,7 @@ static const char *x86_registers[] = {
 	[X86_REG_DR5] = "dr5",
 	[X86_REG_DR6] = "dr6",
 	[X86_REG_DR7] = "dr7",
+#if CS_API_MAJOR >= 4
 	[X86_REG_DR8] = "dr8",
 	[X86_REG_DR9] = "dr9",
 	[X86_REG_DR10] = "dr10",
@@ -103,6 +104,7 @@ static const char *x86_registers[] = {
 	[X86_REG_DR13] = "dr13",
 	[X86_REG_DR14] = "dr14",
 	[X86_REG_DR15] = "dr15",
+#endif
 	[X86_REG_FP0] = "fp0",
 	[X86_REG_FP1] = "fp1",
 	[X86_REG_FP2] = "fp2",
@@ -885,6 +887,12 @@ static RzILOpPure *x86_il_get_operand_bits(X86Op op, int analysis_bits) {
 		break;
 	case X86_OP_MEM:
 		ret = LOADW(BITS_PER_BYTE * op.size, x86_il_get_memaddr_bits(op.mem, analysis_bits));
+		break;
+#if CS_API_MAJOR <= 3
+	case X86_OP_FP:
+		RZ_LOG_WARN("RzIL: x86: Floating point instructions not implemented yet\n");
+		break;
+#endif
 	}
 	return ret;
 }

